components/game_table/dota2/game_table_character_custom.lua

```diff
@@ -0,0 +1,56 @@
+---
+-- @Liquipedia
+-- wiki=dota2
+-- page=Module:GameTable/Character/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Array = require('Module:Array')
+local CharacterIcon = require('Module:CharacterIcon')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+
+local GameTableCharacter = Lua.import('Module:GameTable/Character')
+
+local NONBREAKING_SPACE = '&nbsp;'
+
+local CustomGameTableCharacter = Class.new(GameTableCharacter)
+
+---@return integer
+function CustomGameTableCharacter:getNumberOfBans()
+	return 7
+end
+
+---@param opponentIndex number
+---@param playerIndex number
+---@return string
+function CustomGameTableCharacter:getCharacterKey(opponentIndex, playerIndex)
+	return 'team' .. opponentIndex .. 'hero' .. playerIndex
+end
+
+---@param game CharacterGameTableGame
+---@param opponentIndex number
+---@param key string
+---@return Html
+function CustomGameTableCharacter:_displayCharacters(game, opponentIndex, key)
```

Comment:
if we overwrite it imo remove the leading underscore in the function name (on commons too obviously)